README.md

```diff
@@ -30,71 +32,130 @@ Scrum Master | [@merridy](https://github.com/merridy)
 Product Owner | [@workbygrant](https://github.com/workbygrant)
 
 ## Comms
+
 Slack: LabPlus-team #feijoa
 
 ## Setup
 
+### Prerequisites
+
+- Postgres
+- Java
+- Selenium & ChromeDriver
+
 ### Development
+
 In the application directory:
 
 **1. Make a copy of the example environment file containing some important settings**
 
-```
-> cp example.env .env
+```bash
+cp example.env .env
 ```
 
 **2. Install Rails dependencies and create local databases**
+
+```bash
+bin/setup
 ```
-> bin/setup
-```
 
-**3. [Install and run Elastic search](https://www.elastic.co/downloads/elasticsearch).**
+**3. [Install and run Elastic search](https://www.elastic.co/downloads/elasticsearch)**
+
 By default this is expected to run on localhost:9200 - if you have ElasticSearch running on a different port, you can change `ELASTICSEARCH_URL` in your `.env` file.
 
 On Ubuntu you can use the same script as Travis CI:
-```
+
+```bash
 ELASTIC_SEARCH_VERSION="6.2.3" ./bin/install_elasticsearch.sh
 ```
 
 On Mac you may want to use Homebrew:
-```
+
+```bash
 brew install elasticsearch
 brew services start elasticsearch
 ```
 
+**To check whether Elastic Search is running (or not)**
+`sudo service elasticsearch status`
```

Comment:
Is `sudo` necessary here?

Reply:
Good catch, not needed :stuck_out_tongue: